packages/typespec-test/test/NetworkAnalytics.Management/generated/typespec-ts/sdk/test/arm-test/src/static-helpers/cloudSettingHelpers.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** An enum to describe Azure Cloud. */
+export enum AzureClouds {
+  /** Azure public cloud, which is the default cloud for Azure SDKs. */
+  AZURE_PUBLIC_CLOUD = "AZURE_PUBLIC_CLOUD",
+  /** Azure China cloud */
+  AZURE_CHINA_CLOUD = "AZURE_CHINA_CLOUD",
+  /** Azure US government cloud */
+  AZURE_US_GOVERNMENT = "AZURE_US_GOVERNMENT",
+}
+export function getArmEndpoint(cloudSetting?: AzureClouds): string | undefined {
+  if (cloudSetting === undefined) {
+    return undefined;
+  }
+  const cloudEndpoints: Record<string, string> = {
```

Comment:
small nit for better typing here: 

```suggestion
   const cloudEndpoints: Record<keyof typeof AzureClouds, string> = {
```